src/EleventyWatchTargets.js

```diff
@@ -132,21 +129,16 @@ class EleventyWatchTargets {
 	}
 
 	clearImportCacheFor(filePathArray) {
-		let paths = new Set();
-		for (const filePath of filePathArray) {
-			paths.add(filePath);
-
-			// Delete from require cache so that updates to the module are re-required
-			let importsTheChangedFile = this.getDependantsOf(filePath);
-			for (let dep of importsTheChangedFile) {
-				paths.add(dep);
-			}
+		let paths = new Set([
+			...filePathArray,
```

Comment:
fwiw just as a style preference I try to avoid excessive Array destructuring. I like the previous better here.